sat4envi.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't be commited. Add a line "*.iml" to /.gitignore to avoid commiting it and remove it from the repo in a next commit.

Reply:
done

Reply:
You didn't commit the removal of the file.